easybuild/easyconfigs/s/SIP/SIP-4.19.25-foss-2022a-Python-3.10.4.eb

```
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Bart Verleye
+# Center for eResearch, Auckland
+easyblock = 'ConfigureMakePythonPackage'
+
+name = 'SIP'
+version = '4.19.25'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.riverbankcomputing.com/software/sip/'
+description = """SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['http://sourceforge.net/projects/pyqt/files/sip/sip-%(version)s']
```

I tried to download the file, but it looks like the source URL has been changed for recent versions. The following should work:
```suggestion
source_urls = ['https://www.riverbankcomputing.com/static/Downloads/%(namelower)s/%(version)s']
```

---

Yes, I have noticed that after my pull request. I will add the change soon :)